src/OpenLoco/Utility/String.hpp

```diff
@@ -55,53 +55,53 @@ namespace OpenLoco::Utility
 
     inline size_t strlcpy(char* dest, const char* src, size_t size)
     {
-        size_t src_len = std::strlen(src);
+        size_t srcLen = std::strlen(src);
 
-        if (src_len < size)
+        if (srcLen < size)
         {
-            std::memcpy(dest, src, src_len + 1);
+            std::memcpy(dest, src, srcLen + 1);
         }
         else
         {
             std::memcpy(dest, src, size);
             dest[size - 1] = '\0';
         }
 
-        return src_len;
+        return srcLen;
     }
 
     inline size_t strlcat(char* dest, const char* src, size_t size)
     {
-        size_t src_len = std::strlen(src);
+        size_t srcLen = std::strlen(src);
 
         if (size == 0)
         {
-            return src_len;
+            return srcLen;
         }
 
         // this lambda is essentially a reimplementation of strnlen, which isn't standard
-        size_t dest_len = [=] {
-            auto dest_end = reinterpret_cast<const char*>(std::memchr(dest, '\0', size));
-            if (dest_end != nullptr)
+        size_t destLen = [=] {
+            auto destLen = reinterpret_cast<const char*>(std::memchr(dest, '\0', size));
+            if (destLen != nullptr)
```

Comment:
`dest_end` has been changed into `destLen`, which is the same name as the outer variable. Mistake?

Reply:
it is a mistake but interestingly due to the lambda scope, it's a separate variable. fixed now